### DISCUSSION TITLE
Is 2023 working?  I keep getting Combination not found.

### DISCUSSION THREAD

**Discussion by thekwon333** [TedJuch/TedTournament#13] [General]
=TedTournament("Men",2023,64,32,5)

Combination not found

**Comment by TedJuch** [comment]
You're trying to get the winning team from game 32 in round 64. That's UCLA vs UNC Asheville. It should be returning empty since the winner of that game hasn't been determined yet.

It worked for me. Trying removing the = from the formula, hitting enter to just show the formula, then add the = back and let it load. This will reload the function.
